### PR TITLE
Add a `recompute-text` option to the `many_glyphs` example 

### PR DESCRIPTION
**Pull request by ickshonpe**
# Objective

Add an commandline option for the `many_glyphs` example to force text recomputation every frame.

### CODE REVIEW

File: examples/stress_tests/many_glyphs.rs [hunk 1]
```diff
@@ -71,3 +79,9 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
         ..Default::default()
     });
 }
+
+fn force_text_recomputation(mut text_query: Query<&mut Text>) {
+    for mut text in text_query.iter_mut() {
```
Comment (devil-ira) [inline examples/stress_tests/many_glyphs.rs:84]:
```suggestion
    for mut text in &mut text_query {
```

**Review by devil-ira — APPROVED** [review]

**Review by alice-i-cecile — APPROVED** [review]

**Comment by ickshonpe** [comment]
Okay I'm not sure that was what I meant to do.

**Comment by alice-i-cecile** [comment]
Hmm, that diff doesn't look quite right. TBH may be quicker to just redo this PR from scratch rather than fighting git.

**Comment by ickshonpe** [comment]
Sorry been really busy the last few days, I'll close this PR and create a new one.

**Pull request closed without merging**